src/helpers/measureTitleHelper.ts

```diff
@@ -59,7 +67,8 @@ function findTitleForDerivedMeasure(
     }
     const masterMeasure = getMasterMeasure(measureBucketItems, masterMeasureIdentifier);
     const derivedMeasureTitleBase = getDerivedMeasureTitleBase(masterMeasure);
-    return derivedMeasureTitleBase + suffixFactory.getSuffix(definition);
+    const overTimeComparisonType = findOverTimeComparisonType(definition);
```

Comment:
I suggest renaming `definition` to `definitionType` will be helpful :)

Reply:
Fixed.